### PR TITLE
alignment issue, #298

### PR DESCRIPTION
**Pull request by kshivanku**

### CODE REVIEW

File: client/styles/components/_toolbar.scss [hunk 1]
```diff
@@ -88,6 +88,7 @@
 		}
 	}
 	cursor: pointer;
+	line-height: 18px;
```
Comment (catarak) [inline client/styles/components/_toolbar.scss]: for reasons, I'm not using pixel measurements in this project, but instead `rem`'s and `em`'s. Could you make this
```
line-height: #{18 / $base-font-size}rem;
```

Comment (kshivanku) [inline client/styles/components/_toolbar.scss]: done, sent another PR

**Comment by catarak** [comment]
you can amend commits to the same PR 😄 but i will close this one and merge the other one
